tests/purs/failing/NestedRecordLabelOnTypeError.purs

```diff
@@ -0,0 +1,8 @@
+-- @shouldFailWith TypesDoNotUnify
+module NestedRecordLabelOnTypeError where
+
+record :: { a :: { b :: { c :: Int } } }
+record = { field: 1 }
```

Comment:
```suggestion
record = { a: { b: { c: 1 } } }
```

Reply:
Updated....

Reply:
I thought the intent behind this error was to produce something like:
```
coudl not unify
  Int
with type
  { a :: { b :: { c :: Int } } }

at label `field`
```
Right now, doesn't this test the same thing as one of the other tests?